### DISCUSSION TITLE
Allow setting openai apikey per robot or per user

### DISCUSSION THREAD

**Discussion by phpia** [n4ze3m/dialoqbase#115] [Feature request]
It will be interesting for cost auditing :)

**Comment by n4ze3m** [comment]
This is interesting. I will look into it.

**Comment by n4ze3m** [comment]
Hey, this will be released in next week's update. This week, we focused on the local AI model and chatbot API.

**Comment by phpia** [comment]
Thanks!

**Comment by n4ze3m** [comment]
Version 1.3.0 has been released, which includes the API Key Config per bot feature on the bot settings page. Currently, this only supports chat models and OpenAI models. Other features will be released in the coming weeks.

![image](https://github.com/n4ze3m/dialoqbase/assets/39720973/8a7c23af-e579-40bd-90c8-cda685dfd34a)

**Reply by phpia** [reply]
Thanks I ll test tomorrow
